lib/cookie.js

```diff
@@ -10,6 +10,7 @@ module.exports = class Cookie {
     this.sameSite = cookie.sameSite || null
     this.domain = cookie.domain || null
     this.httpOnly = cookie.httpOnly !== undefined ? cookie.httpOnly : true
+    this.partitioned = cookie.partitioned ?? undefined
```

Comment:
nit: this means this for all practical purposes:

```suggestion
    this.partitioned = cookie.partitioned
```